dockstore-webservice/src/main/java/io/dockstore/webservice/jdbi/WorkflowDAO.java

```diff
@@ -127,6 +138,61 @@ public Workflow findByPath(String path, boolean findPublished) {
         return uniqueResult(query);
     }
 
+    public List<Workflow> findByPaths(List<String> paths, boolean findPublished) {
+        List<Predicate> predicates = new ArrayList<>();
+        SourceControlConverter converter = new SourceControlConverter();
+        Predicate predicate;
```

Comment:
It looks like `predicate` is not used outside of the for look. 
Move the declaration to, say, line 162 to make it clear/easier to read that `predicate` doesn't carry over information between loops.